fix error on AMD CPUs: terminate called after throwing an instance of 'Xbyak::Error' what():  internal error
author: Samuel Moors (Vrije Universiteit Brussel VUB)
diff -ur tensorflow-1.7.0.orig/tensorflow/workspace.bzl tensorflow-1.7.0/tensorflow/workspace.bzl
--- tensorflow-1.7.0.orig/tensorflow/workspace.bzl	2018-03-29 06:18:40.000000000 +0200
+++ tensorflow-1.7.0/tensorflow/workspace.bzl	2018-04-30 16:50:44.336648345 +0200
@@ -105,11 +105,11 @@
   tf_http_archive(
       name = "mkl_dnn",
       urls = [
-          "https://mirror.bazel.build/github.com/01org/mkl-dnn/archive/e0bfcaa7fcb2b1e1558f5f0676933c1db807a729.tar.gz",
-          "https://github.com/01org/mkl-dnn/archive/e0bfcaa7fcb2b1e1558f5f0676933c1db807a729.tar.gz",
+          "https://mirror.bazel.build/github.com/intel/mkl-dnn/archive/v0.13.tar.gz",
+          "https://github.com/intel/mkl-dnn/archive/v0.13.tar.gz",
       ],
-      sha256 = "02e244f63dd95402691a361392504c143eede9a89043426f174836638a9cbf09",
-      strip_prefix = "mkl-dnn-e0bfcaa7fcb2b1e1558f5f0676933c1db807a729",
+      sha256 = "d2cfd93a70cfe86ebe054477c530c9b5c1218b70f75856eb6d1956c68ee89e8f",
+      strip_prefix = "mkl-dnn-0.13",
       build_file = str(Label("//third_party/mkl_dnn:mkldnn.BUILD")),
   )
 
